adflow/pyADflow.py

```diff
@@ -5254,25 +5254,23 @@ def _getSpecialOptionLists(self):
         # pythonOptions do not get set in the Fortran code.
         # They are used strictly in Python.
 
-        pythonOptions = set(
```

Comment:
Why did you change this from a set to a "flat" dictionary?

Reply:
They are the equivalent syntax: `{'a', 'b'} == set(('a', 'b'))`.